src/app/workflow/version-modal/version-modal.component.ts

```diff
@@ -124,4 +124,13 @@ export class VersionModalComponent implements OnInit, AfterViewChecked {
     }
   }
   // Validation ends here
+
+  // Checks if the currently edited test parameter file already exists
+  hasDuplicateTestJson() {
+    if (this.testParameterFilePaths.indexOf(this.testParameterFilePath) > -1) {
```

Comment:
this.testParameterFilePaths.includes(this.testParameterFilePath) might be easier to read.